rocky/rocky/views/mixins.py

```diff
@@ -32,6 +33,7 @@
 
 logger = structlog.get_logger(__name__)
 
+ORIGIN_MAX_AGE = 60*60*24 * 2 # 2 days in seconds
```

Comment:
I suggest using a `timedelta` directly. This way you don't have to create a `timedelta` later on and you get documentation for free.

```suggestion
ORIGIN_MAX_AGE = datetime.timedelta(days=2)
```